client/packages/electron/src/electron.ts

```diff
@@ -59,10 +77,16 @@ const start = (): void => {
     if (answer.name !== SERVICE_NAME) return;
     if (answer.type !== SERVICE_TYPE) return;
 
-    let server: FrontEndHost = JSON.parse(answer.data.toString());
-    server.isLocal = server.ip === ip.address();
+    const server: FrontEndHost = JSON.parse(answer.data.toString());
+    // the IP is showing as 127.0.0.1 for local servers when running on windows
+    server.isLocal = server.ip === ip.address() || server.ip === '127.0.0.1';
 
     window.webContents.send(IPC_MESSAGES.SERVER_DISCOVERED, server);
```

Comment:
should this be in the else below ?

Reply:
actually only if !mruServer.

this way we don't display any servers if alt is not pressed

Reply:
what if the mruServer is set.. but that server isn't found? Then we need the list to show.
I like the idea of not showing if the mruServer is found though.. if that's the only server on the network, we should just connect

Reply:
hmm, good point, i like the change you've done though, thanks